### PR TITLE
First cut at fixing publication bugs

### PR DESCRIPTION
**Pull request by netsettler**
Do a `make build` not `make configure` before running the publication command. (The virtual env has to not just exist but actually contain the right libraries.)

Note that I tested this by attaching a beta (0.3.1.1b0) tag, and the auto-publish worked to that in pypi, so I'm pretty confident this change works. (I've since then de-beta'd as 0.3.2 so this is ready to merge.)

### CODE REVIEW

**Review by dmichaels-harvard — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,12 @@ submitr
 Change Log
 ----------
 
+0.3.2
```
Comment (dmichaels-harvard) [inline CHANGELOG.rst:9]: I think we can/should remove the scripts/publish file as well, no?

Comment (netsettler) [reply CHANGELOG.rst:9]: Yes, I guess so. I'm going to just disable it for now to make sure, but I traced through the call history and it seems GA is (quite indirectly) calling your script from dcicutils.

**Comment by netsettler** [comment]
I also changed around the way modules and submodules headings are done to be a little more aesthetic (better indentation and avoiding some spurious links to subcontent that is not there).

**Comment by netsettler** [comment]
The remaining changes are small so I'll just go ahead and merge. If anyone sees any issues with that, submit a bug report and I'll fix it later.

**Pull request merged**